chainer/functions/array/get_item.py

```diff
@@ -38,7 +38,7 @@ def check_type_forward(self, in_types):
         type_check._argname(in_types, ('x',))
 
     def forward(self, xs):
-        return utils.force_array(xs[0][self.slices]),
+        return utils.force_array(xs[0][backend.from_chainerx(self.slices)]),
```

Comment:
I now feel the attribute fallback https://github.com/chainer/chainer/pull/5745 wasn't robust enough. Don't you think we should extend it to support contained ndarrays too?

Reply:
Beyond the scope of this PR.